src/ir/module-splitting.cpp

```diff
@@ -308,6 +308,9 @@ struct ModuleSplitter {
   // Internal name of the LOAD_SECONDARY_MODULE function.
   Name internalLoadSecondaryModule;
 
+  // Map from original secondary function name to its trampoline
+  std::map<Name, Name> trampolineMap;
```

Comment:
No need for this to be ordered, right?
```suggestion
  std::unordered_map<Name, Name> trampolineMap;
```

Reply:
It looks the trampolines in the tests need to be ordered..?

Reply:
It doesn't have to be ordered but it should be deterministic for the test expectations, but `unordered_map` doesn't seem to guarantee it?